src/Spectre.libGaClassifier/GaFitnessFunction.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+* GaFitnessFunction.cpp
+* It is fitness function for any classifier for GaClassifier.
+*
+Copyright 2018 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "GaFitnessFunction.h"
+#include "Spectre.libException/InconsistentArgumentSizesException.h"
+#include "Spectre.libFunctional/Filter.h"
+#include "Spectre.libClassifier/ConfusionMatrix.h"
+
+namespace spectre::supervised
+{
+
+GaFitnessFunction::GaFitnessFunction(IClassifier& classifier, SplittedOpenCvDataset& data):
```

Comment:
I'd say it's a `ClassifierFitnessFunction`.

It doesn't matter whether `FitnessFunction` is used with GA or other type of optimizer (since we will potentially use others!). What matters is that it scores `IClassifier`.

Reply:
done